cpp/include/cudf/detail/utilities/vector_factories.hpp

```diff
@@ -281,6 +281,20 @@ rmm::device_uvector<typename Container::value_type> make_device_uvector_sync(
   return make_device_uvector_sync(device_span<typename Container::value_type const>{c}, stream, mr);
 }
 
+namespace {
```

Comment:
We try not to use anonymous namespaces in header files.
https://github.com/rapidsai/cudf/blob/branch-0.20/cpp/docs/DEVELOPER_GUIDE.md#anonymous-namespace

Reply:
You are right, good catch.